py-polars/tests/unit/operations/rolling/test_rolling.py

```diff
@@ -818,6 +820,14 @@ def test_rolling_by_date() -> None:
     assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("dtype", [pl.Int64, pl.Int32, pl.UInt64, pl.UInt32])
+def test_rolling_by_integer(dtype: PolarsDataType) -> None:
+    df = pl.DataFrame({"val": [1, 2, 3]}).with_row_index()
```

Comment:
Am I missing something or is the parameterized `dtype` never used?

Shouldn't it be something like this:
```python
df = pl.DataFrame({"val": [1, 2, 3]}, schema={"schema": dtype}).with_row_index()
```